src/main/java/org/capstone/job_fair/services/impl/account/AccountServiceImpl.java

```diff
@@ -49,5 +49,10 @@ public Integer getCountActiveAccountById(String id) {
         return accountRepository.countByIdAndStatus(id, AccountStatus.ACTIVE);
     }
 
+    @Override
+    public Integer getCountActiveAccountByIdAndEmail(String id, String email) {
```

Comment:
Id and email is unique? Why do you need to count using both of them the same time?